api/src/main/java/org/openmrs/module/fhir/api/LocationService.java

```diff
@@ -68,5 +68,12 @@
 	 * @param location representation of location fhir resource
 	 */
 	public void updateLocationById(String id, Location location);
+	
+	/**
+	 * Create location 
+	 *
+	 * @param location representation of location fhir resource
+	 */
+	public Location createLocation(Location location);
```

Comment:
No no, those are not mine. I get a pull --rebase and Sashrika's commits append top of my commits.